src/ui/app.css

```diff
@@ -0,0 +1,115 @@
+:root {
+	/* Theme | vars */
+	--mdc-theme-primary: #4adede;
+	--mdc-theme-primary-dark: #00acac;
+	--mdc-theme-secondary: #787ff6;
+	--mdc-theme-on-primary: #171717;
+	--mdc-theme-on-secondary: white;
+	--mdc-theme-surface: white;
+	--mdc-theme-text-secondary-on-light: rgba(0, 0, 0, 0.6);
+	--mdc-theme-text-primary-on-light: rgba(0, 0, 0, 0.87);
+
+	/* Typography | vars */
+	--mdc-typography-font-family: Roboto, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI',
+		Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+
+	/* body2 */
+	--mdc-typography-body2-font-style: normal;
+	--mdc-typography-body2-font-size: 1em;
+
+	/* body3 */
+	--mdc-typography-body3-font-style: normal;
+	--mdc-typography-body3-font-weight: 400;
+	--mdc-typography-body3-font-size: 0.875em;
+	--mdc-typography-body3-line-height: 143%;
+	--mdc-typography-body3-letter-spacing: 0.17px;
+
+	/* th */
+	--mdc-typography-th-font-style: normal;
+	--mdc-typography-th-font-weight: 500;
+	--mdc-typography-th-font-size: 0.875em;
+	--mdc-typography-th-line-height: 24px;
+	--mdc-typography-th-letter-spacing: 0.17px;
+
+	/* overline */
+	--mdc-typography-overline-font-weight: 400;
+
+	font-family: var(--mdc-typography-font-family);
+}
+
+.mdc-typography--body2,
+.mdc-typography--caption {
+	color: var(--mdc-theme-text-secondary-on-light);
+}
+
+.mdc-typography--body3 {
+	font-style: var(--mdc-typography-body3-font-style);
+	font-weight: var(--mdc-typography-body3-font-weight);
+	font-size: var(--mdc-typography-body3-font-size);
+	line-height: var(--mdc-typography-body3-line-height);
+	letter-spacing: var(--mdc-typography-body3-letter-spacing);
+}
+
+.mdc-typography--th {
+	font-style: var(--mdc-typography-th-font-style);
+	font-weight: var(--mdc-typography-th-font-weight);
+	font-size: var(--mdc-typography-th-font-size);
+	line-height: var(--mdc-typography-th-line-height);
+	letter-spacing: var(--mdc-typography-th-letter-spacing);
+	color: var(--mdc-theme-text-primary-on-light);
+	text-transform: capitalize;
```

Comment:
Non-related discuss with @Madeline-UX for changes to design prior to code reviewing something that is based on the design. Have no problem removing the text-transform capitalize, but it is matching the design that capitalizes the first letter of those. And should be discussed in a later issue rather than delaying prs.

Reply:
@mike-winberry disagree with this flow. We can make an exception for this PR because it's a bit large, but in the future we absolutely should be giving feedback in PRs if something doesn't look right, even if it matches design.  Design is not a line in the sand we must follow, it is a give and take dialog that can change with implementation.  In the future as we have smaller PRs I would expect feedback to given like this by anyone on the team and then the design updated if needed. More importantly, with how we are operating today, there isn't a regularly review of @Madeline-UX's design so she could easily design something that's too hard / not possible if we are providing regular feedback.

Reply:
^ @JasonvanBrackel @YrrepNoj @Noxsios FYSA

Reply:
@mike-winberry probably could have worded it more clearly, but it didn't match the design in the linked issue: https://github.com/defenseunicorns/zarf/issues/802 (potentially it changed?).  I noticed Init (vs. init) in the top bar and figured that was what was trying to be capitalized.

Reply:
Should be fixed now.